src/command/mcl.zig

```diff
@@ -1896,6 +1959,186 @@ fn mclWaitRecoverSlider(params: [][]const u8) !void {
     }
 }
 
+/// Create a logging file for logging register values specified by
+/// the desired register and desired axis. Every successfull setup
+/// can be accessed through log_line variable.
+fn setLogRegisters(params: [][]const u8) !void {
+    const line_name: []const u8 = params[0];
+    const line_idx: usize = try matchLine(line_names, line_name);
+    const line: mcl.Line = mcl.lines[line_idx];
+
+    var log: RegisterLogging = undefined;
+    log.line_index = @intCast(line_idx);
+
+    // Iterate over axis list from parameters
+    var station_indices: []u8 = try allocator.alloc(u8, line.stations.len);
+    var idx: u8 = 0;
+    var axis_input_iterator = std.mem.tokenizeSequence(u8, params[1], ",");
+    outer: while (axis_input_iterator.next()) |token| {
+        const axis_index = try std.fmt.parseInt(mcl.Station.Id, token, 0) - 1;
+        if (axis_index < 0 or axis_index > line.axes.len - 1) {
+            return error.InvalidAxis;
+        }
+        const station_index: Station.Index = @intCast(axis_index / 3);
+        //TODO: Better checking than O(n)
+        for (station_indices) |item| {
+            if (station_index == item) continue :outer;
+        }
+        station_indices[idx] = station_index;
+        idx += 1;
+    }
+    log.station_indices = station_indices[0..idx];
+
+    // Iterate over register list from parameters
+    var reg_input_iterator = std.mem.tokenizeSequence(u8, params[2], ",");
+    while (reg_input_iterator.next()) |token| {
+        if (std.meta.stringToEnum(RegisterLogging.RegisterType, token)) |item| {
+            log.register_lists.set(item, true);
+        } else {
+            return error.InvalidRegister;
+        }
+    }
+
+    var info_buffer: [64]u8 = undefined;
+    const prefix = "Ready to log register: ";
+    @memcpy(info_buffer[0..prefix.len], prefix);
+    var buf_len = prefix.len;
+
+    var register_iterator = log.register_lists.iterator();
+    while (register_iterator.next()) |reg_entry| {
+        if (!log.register_lists.get(reg_entry.key)) continue;
+        @memcpy(info_buffer[buf_len .. buf_len + @tagName(reg_entry.key).len], @tagName(reg_entry.key));
+        @memcpy(info_buffer[buf_len + @tagName(reg_entry.key).len .. buf_len + @tagName(reg_entry.key).len + 1], ",");
+        buf_len += @tagName(reg_entry.key).len + 1;
+    }
+    std.log.info("{s}", .{info_buffer[0 .. buf_len - 1]});
+
+    const path = params[3];
+    var path_buffer: [512]u8 = undefined;
+    const file_path = if (path.len > 0) path else p: {
+        var timestamp: u64 = @intCast(std.time.timestamp());
+        timestamp += std.time.s_per_hour * 9;
+        const days_since_epoch: i32 = @intCast(timestamp / std.time.s_per_day);
+        const ymd =
+            command.chrono.date.YearMonthDay.fromDaysSinceUnixEpoch(days_since_epoch);
```

Comment:
This should not import from `command`, but instead use the `chrono` library directly.